qdrant_client/local/qdrant_local.py

```
@@ -114,6 +115,31 @@ def close(self, **kwargs: Any) -> None:
+        """
+        if self._flock_file is None or self._flock_file.closed:
+            return
+        try:
+            import portalocker  # same import-deferral rationale as `close()`
+            portalocker.unlock(self._flock_file)
+            self._flock_file.close()
+        except (TypeError, Exception):
+            # Same teardown-safety rationale as `close()`: portalocker can
+            # be GC'd before the instance, and any other shutdown error
+            # must not crash interpreter teardown.
+            pass
```

Ruff (0.16.1)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)